androidtv/constants.py

```diff
@@ -448,6 +449,7 @@
         APP_SONY_BRAVIA_TUTORIALS: 'BRAVIA Tutorials',
         APP_SONY_DISCOVER: 'Discover',
         APP_SONY_HELP: 'Help',
+        APP_SONY_LIV: 'SonyLIV',
```

Comment:
Please preserve alphabetical order.

Reply:
This is in alphabetical order, unless you mean something else.